calico-policies/private-network-isolation/eu-es/allow-private-services-pods.yaml

```diff
@@ -52,7 +55,10 @@ spec:
       - 166.9.30.45/32 
       - 166.9.28.159/32 
       - 166.9.32.117/32
-      # End new IP address list
+      # IPs for eu-es
```

Comment:
Are these going to be added to https://cloud.ibm.com/docs/log-analysis?topic=log-analysis-endpoints ?

Reply:
they're there in test.cloud